std/algorithm/searching.d

```diff
@@ -1425,8 +1424,7 @@ if (isInputRange!Range && !isInfinite!Range &&
 }
 
 private auto extremum(alias selector = "a < b", Range)(Range r)
-if (isInputRange!Range && !isInfinite!Range &&
-    !is(typeof(unaryFun!selector(ElementType!(Range).init))))
+if (isInputRange!Range && !isInfinite!Range && !is(typeof(unaryFun!selector(ElementType!(Range).init))))
```

Comment:
This change looks rather restrictive. The constraint is indented properly, but it's split on 2 lines to make it easier to read. Are you sure this is what we want?